src/test/java/uk/gov/hmcts/reform/professionalapi/util/RefDataUtilTest.java

```diff
@@ -309,19 +309,27 @@ public void test_shouldCreatePageableObjectWithDefaultPageSize() {
     }
 
     @Test
-    public void test_getShowDeletedValueTrue() {
-        String showDeleted = "True";
-        String response = RefDataUtil.getShowDeletedValue(showDeleted);
-
-        assertThat(response.equals("true")).isTrue();
+    public void test_getShowDeletedValue() {
+        assertThat(RefDataUtil.getShowDeletedValue("True").equals("true")).isTrue();
+        assertThat(RefDataUtil.getShowDeletedValue("true").equals("true")).isTrue();
+        assertThat(RefDataUtil.getShowDeletedValue("TRUE").equals("true")).isTrue();
+        assertThat(RefDataUtil.getShowDeletedValue("False").equals("false")).isTrue();
+        assertThat(RefDataUtil.getShowDeletedValue("false").equals("false")).isTrue();
+        assertThat(RefDataUtil.getShowDeletedValue("FALSE").equals("false")).isTrue();
+        assertThat(RefDataUtil.getShowDeletedValue("invalid").equals("false")).isTrue();
+        assertThat(RefDataUtil.getShowDeletedValue("").equals("false")).isTrue();
     }
 
     @Test
-    public void test_getShowDeletedValueFalse() {
-        String showDeleted = "false";
-        String response = RefDataUtil.getShowDeletedValue(showDeleted);
-
-        assertThat(response.equals("false")).isTrue();
+    public void test_getReturnRolesValue() {
+        assertThat(RefDataUtil.getReturnRolesValue("True").equals("true")).isTrue();
+        assertThat(RefDataUtil.getReturnRolesValue("true").equals("true")).isTrue();
+        assertThat(RefDataUtil.getReturnRolesValue("TRUE").equals("true")).isTrue();
+        assertThat(RefDataUtil.getReturnRolesValue("False").equals("false")).isTrue();
```

Comment:
can you check for empty space here?

Reply:
This has been done on line 332

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/603/commits/a2eba9506c479fa47b703014a1b64d23efe869d6